chapters/ecosystems/BizLegFoss/Wordpress.md

```diff
@@ -0,0 +1,233 @@
+## Rationale
+Wordpress is a gigantic thing. It powers (at least) 23% of the web.
+(http://w3techs.com/technologies/details/cm-wordpress/all/all) We wanted to
```

Comment:
Could you add text to this link? Even something like \[w3techs](http://w3techs.com/technologies/details/cm-wordpress/all/all) would be fine. No one likes raw links.